### DISCUSSION TITLE
Running "npx prisma db seed" after updating to 3.10.0 returns an error

### DISCUSSION THREAD

**Discussion by larsniet** [prisma/prisma#12015] [Q&A] [answered]
I have been trying to fix the following error: 

```
❯  export DEBUG="prisma*"

❯ npm run db:seed

> nextjs-with-typescript@5.0.0 db:seed
> npx prisma db seed

  prisma:engines binaries to download libquery-engine, migration-engine, introspection-engine, prisma-fmt +0ms
  prisma:loadEnv project root found at /Users/myuser/Documents/SomeFolder/SomeFolder/package.json +0ms
  prisma:tryLoadEnv Environment variables loaded from /Users/myuser/Documents/SomeFolder/SomeFolder/.env +0ms
Environment variables loaded from .env
  prisma:migrate:seed {
  prisma:migrate:seed   prismaConfig: {
  prisma:migrate:seed     data: {
  prisma:migrate:seed       seed: 'ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts'
  prisma:migrate:seed     },
  prisma:migrate:seed     packagePath: '/Users/myuser/Documents/SomeFolder/SomeFolder/.package.json'
  prisma:migrate:seed   }
  prisma:migrate:seed } +0ms
Running seed command `ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts` ...
  prisma:tryLoadEnv Environment variables loaded from /Users/myuser/Documents/SomeFolder/SomeFolder/.env +0ms
  prisma:tryLoadEnv Environment variables loaded from /Users/myuser/Documents/SomeFolder/SomeFolder/.env +8ms
  prisma:client clientVersion: 3.10.0 +0ms
  prisma:client clientEngineType: library +0ms
  prisma:client:libraryEngine internalSetup +0ms
  prisma:client:libraryEngine Searching for Query Engine Library in /Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/.prisma/client +9ms
  prisma:client:libraryEngine loadEngine using /Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/.prisma/client/libquery_engine-darwin-arm64.dylib.node +0ms
  prisma:migrate:seed {
  prisma:migrate:seed   e: Error: Command was killed with SIGKILL (Forced termination): ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts
  prisma:migrate:seed       at makeError (/Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/prisma/build/index.js:3507:17)
  prisma:migrate:seed       at handlePromise (/Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/prisma/build/index.js:4237:33)
  prisma:migrate:seed       at runMicrotasks (<anonymous>)
  prisma:migrate:seed       at processTicksAndRejections (node:internal/process/task_queues:96:5)
  prisma:migrate:seed       at async executeSeedCommand (/Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/prisma/build/index.js:105037:5)
  prisma:migrate:seed       at async Object.parse (/Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/prisma/build/index.js:105147:31)
  prisma:migrate:seed       at async main (/Users/myuser/Documents/SomeFolder/SomeFolder/.node_modules/prisma/build/index.js:108740:18) {
  prisma:migrate:seed     shortMessage: 'Command was killed with SIGKILL (Forced termination): ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts',
  prisma:migrate:seed     command: 'ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts',
  prisma:migrate:seed     escapedCommand: 'ts-node --compiler-options "{\\"module\\":\\"CommonJS\\"}" "prisma/seed.ts"',
  prisma:migrate:seed     exitCode: undefined,
  prisma:migrate:seed     signal: 'SIGKILL',
  prisma:migrate:seed     signalDescription: 'Forced termination',
  prisma:migrate:seed     stdout: undefined,
  prisma:migrate:seed     stderr: undefined,
  prisma:migrate:seed     failed: true,
  prisma:migrate:seed     timedOut: false,
  prisma:migrate:seed     isCanceled: false,
  prisma:migrate:seed     killed: false
  prisma:migrate:seed   }
  prisma:migrate:seed } +2s

An error occured while running the seed command:
Error: Command was killed with SIGKILL (Forced termination): ts-node --compiler-options {"module":"CommonJS"} prisma/seed.ts
```

The error happens, as shown, when trying to seed the database. I have not changed anything in the prisma schema, only updated to the latest version. Before the update everything was working fine. 

This is the seeder: 
```
import { hash } from "bcryptjs";
import { domains } from "./seeds/domains";
import { PrismaClient } from "@prisma/client";

const prisma = new PrismaClient();

async function main() {
	const encryptedPassword = await hash("password", 12);

	// Create users
	await Promise.all([
		prisma.user.upsert({
			where: { email: "john@doe.eu" },
			update: {},
			create: {
				name: "John Doe",
				email: "john@doe.eu",
				password: encryptedPassword,
				role: "ADMIN",
			},
		}),
	]);

	// Create domains without owner
	await Promise.all(
		domains.map(async (domain) =>
			prisma.domain.upsert({
				where: { url: domain.url },
				update: {},
				create: domain,
			})
		)
	);
}

main()
	.catch((e) => {
		console.error(e);
		process.exit(1);
	})
	.finally(async () => {
		await prisma.$disconnect();
	});
```

Some additional info:
- Project is build on NextJS
- Prisma is connected to a PostgreSQL database running on Ubuntu 20.0
- I have tried almost all commands from Prisma, including migrate reset, migrate dev and db push (they all work fine)
- I checked the database which shows that all tables are empty

If anything else is needed, let me know :). Thanks in advance.

**Comment by larsniet** [comment, marked as answer]
Turns out, it was a node modules thing (as always 😞). Resolved the issue by removing node_modules folder and package-lock.json file and reinstalling using ```npm install```.

**Reply by dgyurics** [reply]
This helped. Thanks

**Reply by binjuhor** [reply]
Thank you this helped.

**Reply by SilasRodrigues19** [reply]
Thank you @larsniet, it solved

**Reply by Evangeliont** [reply]
Thanks!

**Comment by eran-or** [comment]
same happens with remix.run . it should be a bug and not discussion

**Reply by janpio** [reply]
Then someone needs to create a bug issue and fill the issue template, which optimally includes a reproduction so we can reproduce this problem and look at how fixing this.

**Comment by taylor-lindores-reeves** [comment]
As per @larsniet & @dgyurics answers, deleting node_modules and lock file and rerunning seed command fixed the issue for me.

**Comment by Suraj-Kumar-Hemnani** [comment]
The solution suggested by @larsniet didn't work for me. The issue is with the script that is written. Try the following script:

```  
"seed": "ts-node --compiler-options \"{\\\"module\\\":\\\"commonjs\\\"}\" --transpile-only prisma/seed.ts",
   ```

**Reply by magicspon** [reply]
this works when using `yarn`, but it's failing when using `pnpm`

**Comment by nojitsi** [comment]
reinstalled node_modules - not helped.
restarted mac after - helped.

**Comment by top1st** [comment]
I think the problem is related with openssl version on ubuntu 20.04
```
/node_modules/.prisma/client/libquery_engine-debian-openssl-1.1.x.so.node +2ms
  prisma:client:libraryEngine  library starting +56ms
Segmentation fault
```

**Reply by janpio** [reply]
Yours might be, but it is also different from all the other reports here. So best open a new discussion where you describe your full problem.

**Comment by AlbinXXX** [comment]
![image](https://github.com/prisma/prisma/assets/136234404/539879fb-5eb8-4915-8fc8-ecd011df8b0d)
![image](https://github.com/prisma/prisma/assets/136234404/240d6d54-4301-49fd-bbec-21c45a980df2)

I've tried everything at this point :/

**Reply by janpio** [reply]
You are generating to a custom `output` under `./prisma/generated/client` but are probably just importing `@prisma/client`, instead of trom that path. Possible?

**Comment by tigerrabbit** [comment]
Here is an example that works for me using drizzle instead of prisma with a similar issue.  Note this is on nextjs with vercel postgres which uses neon (serverless postgres) behind the scenes.  In package.json scripts section
`    "db:seed": "ts-node --compiler-options '{\"module\":\"CommonJS\"}' ./src/lib/seed.ts"

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
